core/pbcc_revoke_token.h

```diff
@@ -0,0 +1,17 @@
+/* -*- c-file-style:"stroustrup"; indent-tabs-mode: nil -*- */
+// #if !defined INC_PBCC_GRANT_TOKEN_API
```

Comment:
@650elx should this be uncommented and different name used?

Reply:
indeed; that was a temp solution